data/src/new_etl/data_utils/vacant_properties.py

```diff
@@ -105,17 +107,26 @@ def vacant_properties(primary_featurelayer: FeatureLayer) -> FeatureLayer:
             vacant_properties.gdf["parcel_type"] != "Land"
         ]
 
-        # Load backup data
-        backup_gdf = load_backup_data_from_gcs("vacant_indicators_land_06_2024.geojson")
+        # Attempt to load backup data from GCS
+        try:
+            backup_gdf = load_backup_data_from_gcs(
+                "vacant_indicators_land_06_2024.geojson"
+            )
 
-        # Add parcel_type column to backup data
-        backup_gdf["parcel_type"] = "Land"
+            # Add parcel_type column to backup data
+            backup_gdf["parcel_type"] = "Land"
 
-        # Append backup data to the existing dataset
-        print(f"Appending backup data ({len(backup_gdf)} rows) to the existing data.")
-        vacant_properties.gdf = pd.concat(
-            [vacant_properties.gdf, backup_gdf], ignore_index=True
-        )
+            # Append backup data to the existing dataset
+            print(
+                f"Appending backup data ({len(backup_gdf)} rows) to the existing data."
+            )
+            vacant_properties.gdf = pd.concat(
+                [vacant_properties.gdf, backup_gdf], ignore_index=True
+            )
+        except Exception as e:
```

Comment:
Something to add an issue for -- we probably want to add an issue to refactor these `except Exception as e` lines. I think we generally want to try to be more specific with error catching, as it usually helps debug issues. Not something to address in this PR though.